CA2025 warnings setting net10.0 target framework

```
/Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/ClientConnection.cs(508,59): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/ClientConnection.cs(532,56): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/ClientConnection.cs(537,59): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/ConnectionCache.cs(372,59): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/ConnectionCache.cs(394,71): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/ConnectionCache.cs(399,59): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc/Server.cs(487,76): error CA2025: Ensure tasks using 'IDisposable' instances complete before the instances are disposed (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2025)
```

--- a/build/CodeAnalysis.Base.globalconfig
+++ b/build/CodeAnalysis.Base.globalconfig
@@ -103,7 +103,7 @@ dotnet_diagnostic.CA2015.severity = warning
 dotnet_diagnostic.CA2016.severity = warning
 
 # CA2025: Do not pass 'IDisposable' instances into unawaited tasks
-# We often pass Task (an IDisposable) to fire-and-forget async methods that take ownership of their lifetime.
+# We occasionally pass a Task (an IDisposable object) to an un-awaited async method that awaits this task.
 dotnet_diagnostic.CA2025.severity = none
 
 # CA2200: Rethrow to preserve stack details

